opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/exponential_histogram/buckets.py

```diff
@@ -73,6 +73,10 @@ def index_base(self, value: int) -> None:
     def counts(self):
         return self._counts
 
+    def get_offset_counts(self):
+        bias = self.__index_base - self.__index_start
+        return self._counts[-bias:] + self._counts[:-bias]
```

Comment:
collected result should be a regular list, not a circular buffer, so introduce a new method for it